ddlitlab2024/dataset/pytorch.py

```diff
@@ -339,12 +366,14 @@ def __getitem__(self, idx: int) -> Result:
     def collate_fn(batch: Iterable[Result]) -> Result:
         return DDLITLab2024Dataset.Result(
             joint_command=torch.stack([x.joint_command for x in batch]),
-            joint_command_history=torch.stack([x.joint_command_history for x in batch]),
-            joint_state=torch.stack([x.joint_state for x in batch]),
-            image_data=torch.stack([x.image_data for x in batch]),
-            image_stamps=torch.stack([x.image_stamps for x in batch]),
-            rotation=torch.stack([x.rotation for x in batch]),
-            game_state=torch.tensor([x.game_state for x in batch]),
+            joint_command_history=torch.stack([x.joint_command_history for x in batch])
```

Comment:
I would always prefer the usage of empty collections instead of `null|None` whenever possible to prevent issues with NullPointers and be able to use consistent collection APIs without needing to rely on null checks.
I'm unsure of the context here though. Would an empty `torch.stack([])` break training?